src/main/java/io/managed/services/test/client/kafkainstance/KafkaInstanceApiUtils.java

```diff
@@ -54,19 +52,22 @@ public static KafkaInstanceApi kafkaInstanceApi(KafkaRequest kafka, String offli
     }
 
     public static KafkaInstanceApi kafkaInstanceApi(String uri, String offlineToken) {
-        ApiClient client = new ApiClient();
-
-        if (Environment.KAFKA_INSECURE_TLS) {
-            ClientConfiguration clientConfig = new ClientConfiguration(ResteasyProviderFactory.getInstance());
-            clientConfig.register(client.getJSON());
-
-            client.setHttpClient(ClientBuilder.newBuilder()
-                    .sslContext(TestUtils.getInsecureSSLContext("TLS"))
-                    .withConfig(clientConfig)
-                    .build());
-        }
-
-        return new KafkaInstanceApi(client.setBasePath(uri), offlineToken);
+        var adapter = new OkHttpRequestAdapter(new BaseBearerTokenAuthenticationProvider(new RHAccessTokenProvider(offlineToken)));
+        adapter.setBaseUrl(uri);
+        ApiClient client = new ApiClient(adapter);
+
+//        TODO: should this be allowed?
```

Comment:
@henryZrncik have you taken a look at this TODO? wdyt?

Reply:
still TODO, there were clusters which had this server set as http, so we kept it like that. But this was not case for quite a long time.